src/app/routes/topic/getInitialData/index.js

```diff
@@ -12,35 +13,37 @@ const popId = path => path.split('/').pop();
 
 const getId = pipe(getUrlPath, removeAmp, popId);
 
-export default async ({ getAgent, service, path: pathname, variant }) => {
+export default async ({ getAgent, service, path: pathname, variant, page }) => {
   try {
     const env = pathname.includes('renderer_env=test') ? 'test' : 'live';
     const agent = await getAgent();
     const id = getId(pathname);
-    const path = `${process.env.BFF_PATH}?id=${id}&service=${service}${
-      variant ? `&variant=${variant}` : ``
-    }`;
-
+    const parsedBffUrl = Url(process.env.BFF_PATH).set('query', {
+      id,
+      service,
+      ...(variant && {
+        variant,
+      }),
+      ...(page && {
+        page,
+      }),
+    });
     const optHeaders = { 'ctx-service-env': env };
     const { status, json } = await fetchPageData({
-      path,
+      path: parsedBffUrl.toString(),
       agent,
       optHeaders,
     });
     const { data } = json;
+
     return {
       status,
       pageData: {
         title: data.title,
         description: data.description || data.title,
         promos: data.summaries,
-
-        // TODO - update me when the BFF is returning this data
         activePage: data.activePage || 1,
-
-        // TODO - update me when the BFF is returning this data
         pageCount: data.pageCount || 99,
```

Comment:
BFF should always return this now.

If something goes wrong and it doesn't include this, having no pagination is probably a better fallback than having the component show there being 99 pages
```suggestion
        pageCount: data.pageCount,
```

Reply:
Yeh I did wonder about removing the fallback value. Cheers!